core/chains/evm/txmgr/models.go

```diff
@@ -26,8 +27,25 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/utils"
 )
 
-type EvmReceipt = txmgrtypes.Receipt[evmtypes.Receipt, common.Hash]
-type EvmReceiptPlus = txmgrtypes.ReceiptPlus[evmtypes.Receipt]
+// Type aliases for EVM
+type EvmEthConfirmer = EthConfirmer[*evmtypes.Address, *evmtypes.TxHash, *evmtypes.BlockHash]
+type EvmEthBroadcaster = EthBroadcaster[*evmtypes.Address, *evmtypes.TxHash, *evmtypes.BlockHash]
+type EvmEthResender = EthResender[*evmtypes.Address, *evmtypes.TxHash, *evmtypes.BlockHash]
```

Comment:
Can we avoid these `EvmEth` double prefixes?
```suggestion
type EVMConfirmer = Confirmer[*evmtypes.Address, *evmtypes.TxHash, *evmtypes.BlockHash]
type EVMBroadcaster = Broadcaster[*evmtypes.Address, *evmtypes.TxHash, *evmtypes.BlockHash]
type EVMResender = Resender[*evmtypes.Address, *evmtypes.TxHash, *evmtypes.BlockHash]
```

Reply:
We do have a plan to rename all Eth* and remove the Eth prefix, towards the end of this work.
But this is a fair point, that the name EvmEth* can right now be changed to Evm*.
Doing it now.

Reply:
also would it be cleaner to create an `alias.go` file rather than have these all inside `models.go`